pycds/__init__.py

```diff
@@ -364,7 +365,7 @@ class ObsWithFlags(Base):
 sqlalchemy.event.listen(
     metadata, 'before_create',
     DDL('''
-        CREATE OR REPLACE FUNCTION DaysInMonth(date) RETURNS double precision AS
+        CREATE OR REPLACE FUNCTION crmp.DaysInMonth(date) RETURNS double precision AS
```

Comment:
Oh, except that tests can't access something in the prod database. Awesomeness lost.